src/app/_mermaid_common/libs/services/offline-table-backup.svc.js

```diff
@@ -20,7 +21,7 @@ angular.module('mermaid.libs').service('OfflineTableBackup', [
 
     var getProjectName = function(projectId) {
       return offlineservice
-        .ProjectsTable(true)
+        .ProjectsTable(projectId)
```

Comment:
Is this really right?

Reply:
I copied that from Dustin's pr https://github.com/data-mermaid/mermaid-collect/pull/20 as he referred to this assuming that pr may be pushed later on. So I think there won't be a conflict